src/scene/layer.js

```diff
@@ -43,51 +42,22 @@ let layerCounter = 0;
 
 const lightKeys = [];
 
-class VisibleInstanceList {
-    constructor() {
-        this.list = [];
-        this.length = 0;
-        this.done = false;
-    }
-}
-
-class InstanceList {
-    constructor() {
-        this.opaqueMeshInstances = [];
-        this.transparentMeshInstances = [];
-        this.shadowCasters = [];
-
-        // arrays of VisibleInstanceList for each camera of this layer
-        this.visibleOpaque = [];
-        this.visibleTransparent = [];
-    }
-
-    // prepare for culling of camera with specified index
-    prepare(index) {
-
-        // make sure visibility lists are allocated
-        if (!this.visibleOpaque[index]) {
-            this.visibleOpaque[index] = new VisibleInstanceList();
-        }
-
-        if (!this.visibleTransparent[index]) {
-            this.visibleTransparent[index] = new VisibleInstanceList();
-        }
+const _tempMaterials = new Set();
 
-        // mark them as not processed yet
-        this.visibleOpaque[index].done = false;
-        this.visibleTransparent[index].done = false;
-    }
+class CulledInstances {
```

Comment:
Does this class need an `@ignore` tag?

Reply:
It should not as it does not have a JSDoc block for it.